dmutils/email/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Allow accessing send email from email. Backwards compatibility."""
+from dmutils.email.dm_mandrill import send_email
```

Comment:
The only reason I didn't bother was that it should make refactoring a bit easier if we have explicit import. This is only a backwards compatibility inclusion. Not _needed_ for new code. Also we'd have to import the client not the method then it might look a bit weird getting the client for Notify and the send method for Mandrill from the same place :/